### ISSUE TITLE
Your website is down

### ISSUE THREAD

**Issue by edbird** [omaflak/Asynchronous-Socket-Class-C-Windows#5]
See title

**Comment by omaraflak** [comment]
Indeed, I have shut down the website. It won't be available anymore unfortunately.

You can access whatever is left using web.archive.org though: https://web.archive.org/web/20200811124244/http://causeyourestuck.io/

**Comment by edbird** [comment]
I only ask because I'm interested in porting your software to work on Linux. If I get anywhere with it I can send you a PR. At the moment I'm struggling to find the definition of things like `SOCKET` on Windows. I'm not really a windows user myself you see.

Do you by any chance know what reference you used to build on the winsock32 stuff?

**Comment by omaraflak** [comment]
Yeah sockets on windows are kind of messy. I did make another repo for sockets on UNIX systems though: https://github.com/omaraflak/async-socket

However, this is more for educational purposes rather than actual production code... If you need a nice library take a look at socket.io, I believe they have a C++ client, don't know for the server though.
